framework/doc/resources.rst

```diff
@@ -66,10 +50,10 @@ This example could be enhanced to dynamically react to bundles being
 started and stopped, making use of the popular *extender pattern* from
 OSGi.
 
-Runtime overhead
+Runtime Overhead
 ----------------
 
-The resources system has the following runtime characteristics:
+The resource has the following runtime characteristics:
```

Comment:
"resource" doesn't seem to be the right word considering what follows describes the entire resource system as a whole.